lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -1,16 +1,28 @@
+require "pathname"
+
 module CC
   module Analyzer
     class IssueRelativePathValidation < Validation
       def valid?
-        path && !path.start_with?("/")
+        path &&
+          !path.start_with?("/") &&
+          File.exist?(path) &&
```

Comment:
Ah that's right. I forgot about that validation. I'll remove this `exist?` check.